### DISCUSSION TITLE
Proposed migrations: Jitsi* and ProtonMail/Cal and Element/Matrix

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#384] [Ideas]
These products prioritize security and/or Open Source. 

Proton offers 10 emails, and up to three custom domains, so we can have 8-9 OMI group emails if we want all for a total of $11.99/mo. 

I'd like to see us migrate to more open source and privacy-focused tools. Ultimately, I think we could shift away from discord and github too. 

I can demo both during a future Wednesday call.

**Comment by mrmetaverse** [comment]
Cc: @antpb can we add this to the upcoming agenda?  

Cc: @humbletim maybe this could work for our previous email needs

**Comment by fire** [comment]
It is unclear that these proprietary services are better than self hosting, or gmail, and for example self hosting email is a nightmare.

We can selfhost video and voice chat or use https://meet.jit.si

Our status quo is that meetings are recorded, so it doesn't request privacy concerns.

**Comment by fire** [comment]
The main purpose of Github is its reach as a social network. I've had great experience with https://gitlabhost.com/ (using their community edition instead of enterprise) but it is not recommend because such a site does not have the reach or impact.

**Comment by madjin** [comment]
Highly in favor of proposals like this that inch us closer to becoming more digitally sovereign. Protonmail and Jitsi are good, but I wonder if there's a Matrix solution out there worth considering? @robertlong

**Comment by mrmetaverse** [comment]
> Highly in favor of proposals like this that inch us closer to becoming more digitally sovereign. Protonmail and Jitsi are good, but I wonder if there's a Matrix solution out there worth considering? @robertlong

Agreed, I'd like to switch everything over to matrix/element and understand what some may see as a "conflict of interest" considering our co-chairs affiliation, but through my research it seems to be the best option. Robert has suggested waiting for a few key features before the migration will be more demonstrably worth the effort. 

We actually reserved both OMI and openmetaverse in matrix, and can migrate any time. But tools like jitsi and protonmail solve for different issues, like how we cast our meetings, and how we field receipts/news/updates and calendar. 

I guess a loose migration plan looks: 

Activity | Current | Migrate to _something like this_
---|---|---
Email and Calendar | gsuite | -> Proton (Mail, Cal, Drive)
Chat and Channels | discord | -> element / matrix
Documentation and Issues | Github | -> Gitlab
Streaming and Video Calls | Meet and Discord | -> Jitsi (continue to be bridged to youtube and streamyard)
Workshops and Brainstorming | miro | -> tldraw

**Comment by mrmetaverse** [comment]
Rocketchat is a free open source privately hosted text chat. This is what godot uses. Could possibly be integrated with discord.

**Comment by mrmetaverse** [comment]
Big Blue Button - alternative for meetings and conferences

**Comment by mrmetaverse** [comment]
I would suggest moving to gitlab, but its a per-user charge for feature parody, and that is not a wise move for our open model. Unless I am missing something here. 

I propose migrating to a gitea service, either self-hosting or using something like codeberg.

**Comment by mrmetaverse** [comment]
So... what can we do now? Do we want to migrate from google to proton? Or start using element instead of discord? Or start streaming using youtube live and jitsi and matrix, etc.? 

## How might we make this actionable today?

**Comment by mrmetaverse** [comment]
![image-4.png](https://user-images.githubusercontent.com/63426722/205417428-c4b8f905-a793-480f-980d-abed648a5c62.png)

**Comment by madjin** [comment]
## Proton > Gsuite

This deal goes on through end of the year, it will cost about $172 for 24 months for 3 custom email domains (@omigroup.org for example) and up to 15 email addresses (like an alias). This will save admin time from having to think about this stuff every month, saving us untold amounts of stress / administrative overhead costs.

Proton supports lots of good payment options too, and it will give us awesome gsuite like features such as 500gb of space. It's more private than Google and easy to import.

![image](https://user-images.githubusercontent.com/32600939/205417854-cacd3969-9281-46a1-8b9c-91376f8b09c7.png)

## tldraw > Miro

Currently Miro is costing us $40 a month and doesn't seem like it's worth the money, there's an open source project called [tldraw](https://github.com/tldraw/tldraw) that has many of the same features and MIT licensed.

**Comment by fire** [comment]
The privacy of self hosting and fear of corporations is not the problem we're solving.

From our months of omigroup champion meetings the time that is spent was done on administrative collection of receipts for open collective. That has been many workhours of multiple people's times.

My proposal is to pick a choice Miro? 1password? Gsuite? protonmail? Blog? etc. Any paid service for a long period of time like a year.

Jin and humbletim have discussed with me.

**Comment by mrmetaverse** [comment]
I move to sign up for proton for 2 yrs 

I move to reduce miro to 1 seat or less and adopt the proposed alternative `tldraw` 

The cc payments are a recurring friction/frustration so I support your proposal to reduce the monthlies. 

Should we vote or make the cancels/purchase?

**Comment by madjin** [comment]
I say we budget $300 for 2 years of email + proton services- $200 for proton and $100 for the person helping setup and administrate: https://protonvpn.com/blackfriday/#cards

**Comment by fire** [comment]
1. as far as I know the main service we're providing through gmail is the MSF emails.
2. gsuite is an industry standard like office 365. proton mail is not standard.

**Comment by fire** [comment]
The essential service we're providing and causes OMIGroup problems if disrupted is the email service.

Miro and Figjam have been great events but aren't essential to the services we've providing.

Changing from an gmail address to an protomail address is adminstrative work. What are we gaining?

What is the cost to provide gmail for every partipating member of omigroup in chunks of a year?

**Comment by mrmetaverse** [comment]
Ill add that open source includes an ethos. Often when I get feedback from those in the private sector they say "you have to prove to founders and funders alike that open source and collaborative standards outweigh the value propositions of electing to build a walled garden." I believe this is true. 

I think adopting private (non-data harvesting) and/or open source alternatives helps _show_ instead of tell. Maybe I'm being grandiose but I'm happy to see practical viability, and feasibility use cases driving the decision as well. 

Just beware, we do generally sacrifice some fidelity of usability. I am generally ok with this for myself, and have tested both of the suggested changes in our previous comments so I am confident they will do more than suffice.

**Comment by madjin** [comment]
Isn't g-suite costing lots of administrative overhead without bulk payment options?

**Comment by fire** [comment]
https://apps.google.com/supportwidget/articlehome?hl=en&article_url=https%3A%2F%2Fsupport.google.com%2Fa%2Fanswer%2F1247364%3Fhl%3Den&product_context=1247364&product_name=UnuFlow&trigger_context=a

**Comment by mrmetaverse** [comment]
> https://apps.google.com/supportwidget/articlehome?hl=en&article_url=https%3A%2F%2Fsupport.google.com%2Fa%2Fanswer%2F1247364%3Fhl%3Den&product_context=1247364&product_name=UnuFlow&trigger_context=a

![Screenshot_20221202-223325.png](https://user-images.githubusercontent.com/63426722/205420752-9e3ad073-404e-4acf-8af8-c5e718a3aca9.png)

🫠

**Comment by mrmetaverse** [comment]
> The essential service we're providing and causes OMIGroup problems if disrupted is the email service.
> 
> Miro and Figjam have been great events but aren't essential to the services we've providing.
> 
> Changing from an gmail address to an protomail address is adminstrative work. What are we gaining?
> 
> What is the cost to provide gmail for every partipating member of omigroup in chunks of a year?

I appreciate the thoughtful questions. 

Scalability without needing to add more seats for a little while. Scalability for domains and potential tangential efforts, maybe we never need it, but its included. 

And the storytelling aspect is worth something imo. Showing that alternatives exist outside of the many duopolies. 

And cost

**Comment by fire** [comment]
![image](https://user-images.githubusercontent.com/32321/205424953-a06ee7c4-4545-4794-9adb-e36969b41fb4.png)

**Comment by mrmetaverse** [comment]
Most practical: 
- I recommend we cancel our Miro and switch to tldraw. 
- any paid services we should strive to pay for longer periods (2 years if possible) 
- switching from gsuite to proton would require training people. There are some conflicts like "there aren't any google docs or sheets equivalents"

**Comment by mrmetaverse** [comment]
converting to a discussion.
